datasets/us/dea_fugitives/crawler.py

```diff
@@ -0,0 +1,75 @@
+from zavod import Context, helpers as h
+
+
+def parse_table(table):
+    """This function is used to parse the table of Labels and Descriptions
+    about the fugitive and return as a dict.
+    """
+
+    info_dict = {}
+
+    # The first row will always be the header (Label, Description)
+    # So we can skip it.
+    for row in table.findall(".//tr")[1:]:
+        label = row.findtext(".//td[1]")
+        description = row.findtext(".//td[2]")
+        info_dict[label] = description
+
+    return info_dict
+
+
+def crawl_item(fugitive_url: str, context: Context):
+    response = context.fetch_html(fugitive_url)
+
+    name = response.findtext('.//h2[@class="fugitive__title"]')
+    info_dict = parse_table(response.find(".//table"))
+
+    entity = context.make("Person")
+    entity.id = context.make_id(name)
+
+    if "Sex" in info_dict:
+        if info_dict["Sex"] == "Male":
+            entity.add("gender", "male")
+        else:
+            entity.add("gender", "female")
```

Comment:
what are the values they provide? I think it's better to see if we can automatically normalise and accept these values, and add [lookup mappings](https://zavod.opensanctions.org/best_practices/datapatch_lookups/) for those we can't

Reply:
have a look how I addressed this in https://github.com/opensanctions/opensanctions/commit/745c748a59ac07f4ee636af251cabc2f01e14cc0 - we prefer mappings like this over python logic since it's clear once you know the approach and easy to add more cases.

Reply:
Nice, I will implement this pattern in the next crawlers